src/main/java/core/basesyntax/service/Parser.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+
+public interface Parser {
+    List<FruitTransaction> parseTransactions(List<String> lines);
```

Comment:
Better to specify model in class name as we may have multiple parsers in our application and this is standard to have separate parser for each one
```suggestion
public interface TransactionParser {
    List<FruitTransaction> parseAll(List<String> lines);
```